### PR TITLE
Info about getting a log message to appear in a newly created plugin

### PR DESCRIPTION
**Pull request by uglycoyote**
see #644 

when getting started on a new plugin for the first time, one needs a way to verify that it works, and printing one line to a log seems like a good start.   I wanted to update the docs to make this a bit more straightforward since I stumbled on this more than I should have when trying to get things rolling.

### CODE REVIEW

**Comment by phit** [comment]
maybe add `logger.info` (this will show in console) too?

**Comment by uglycoyote** [comment]
@phit  that is good.  I didn't realize info's would come to the console and debug's would not.  I can remove the big "Tip" that I added to the page then.

**Comment by ryantheleach** [comment]
Nice update! It's good as-is, but a potential improvement could be made that you wouldn't be aware about. (and the previous example wasn't any better in this regard)

Code examples don't get translated through our translation system, in order to prevent symbol names changing.

So a soft rule of thumb the docs team has, is that if a comment in the code is meaningful, it should also be restated outside of the code block so that it's translate-able.

**Comment by uglycoyote** [comment]
@ryantheleach thanks, based on this  I decided to remove the long comments and add a bit of extra text above the code sample.  I guess the translation is automated?  I noticed when browsing the documentation auf Deutsch that the "Note" and "Tip" highlighted sections also remain in English, which seems strange.   I wonder if the translation mechanism has some settings that can be tweaked to get those to translate as well?

**Comment by ST-DDT** [comment]
The translation is done by our translators and is not automated. German currently has a translation coverage of about 60%. So there might be entire pages that still have to be translated but the community is on it. AFAICT the information boxes are translateable as well (see the index page) but that particular might not have been translated yet.

**Comment by ryantheleach** [comment]
The specific part you are talking about, Note vs Tip, is probably due to those being a part of the theme.

**Comment by uglycoyote** [comment]
It was the contents of the Notes and Tips that was not translated.  But looking at other languages I saw that the Notes and Tips of those were translated.  As @ST-DDT says, it's a manual process so I guess the translators have not gotten to those parts for whatever reason.  I'm impressed that there's manual translation!

**Review by Inscrutable — APPROVED** [review]

File: source/plugin/plugin-class.rst [hunk 1]
```diff
@@ -62,21 +62,32 @@ interacting with the game, such as registering commands or events.
 
 Your plugin can listen for particular events, called **state events**, to be notified about changes in the state of the
 game. In the example below, ``onServerStart()`` is called when the :javadoc:`GameStartedServerEvent` occurs; take note
-of the :javadoc:`Listener` annotation before the method.
+of the :javadoc:`Listener` annotation before the method.  
+
+The example below will log a message upon starting the server.  If your plugin is correctly loaded, you should see this message as part of the server's initialization output.
```
Comment (12awsomeman34) [inline source/plugin/plugin-class.rst]: An extra space before If

File: source/plugin/plugin-class.rst [hunk 2]
```diff
@@ -62,21 +62,32 @@ interacting with the game, such as registering commands or events.
...
+        @Inject
+        private Logger logger;
+
         @Listener
-        public void onServerStart(GameStartedServerEvent event) {
-            // Hey! The server has started!
-            // Try instantiating your logger in here.
-            // (There's a guide for that)
+        public void onServerStart(GameStartedServerEvent event)
+        {
```
Comment (12awsomeman34) [inline source/plugin/plugin-class.rst]: Minor, but most examples follow the Sponge guidelines and leave the `{` on the same line as the function

**Review by 12awsomeman34 — CHANGES_REQUESTED** [review]
Minor stuff really...

File: source/plugin/plugin-class.rst (hunk 1 above)
Comment (12awsomeman34) [reply source/plugin/plugin-class.rst]: Looking at this again, it's a really long line that should be divided into two, ref: https://github.com/SpongePowered/SpongeDocs/pull/725#discussion_r205389138

**Comment by uglycoyote** [comment]
I wrapped the long lines and moved the curly brace to conform with the coding standard.

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/568431c/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/568431c/plugin/plugin-class.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/724#issuecomment-408175138), not to the direct url to the preview._

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
@NariKiro Can you review this again? Are all issues fixed so we can merge this?

File: source/plugin/plugin-class.rst [hunk 3]
```diff
@@ -62,28 +62,39 @@ interacting with the game, such as registering commands or events.
 
 Your plugin can listen for particular events, called **state events**, to be notified about changes in the state of the
 game. In the example below, ``onServerStart()`` is called when the :javadoc:`GameStartedServerEvent` occurs; take note
-of the :javadoc:`Listener` annotation before the method.
+of the :javadoc:`Listener` annotation before the method.  
+
+The example below will log a message upon starting the server.  If your plugin is correctly loaded,
```
Comment (12awsomeman34) [inline source/plugin/plugin-class.rst]: Still two spaces before If in this sentence, but that has no effect on the preview itself. Everything else is fine.

**Review by 12awsomeman34 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by uglycoyote** [comment]
@inscrutable thanks for fixing the space problem.  Sorry was out of touch for a bit on this one, been doing some handyman projects that took up all my free time.

**Pull request merged**